cub/cub/util_type.cuh

```diff
@@ -44,6 +44,7 @@
 #endif // no system header
 
 #include <cub/detail/uninitialized_copy.cuh>
+#include <cub/util_deprecated.cuh>
```

Comment:
Why is that additional include needed? Should it be
```suggestion
#include <cub/util_arch.cuh>
```

Reply:
It's a drive-by fix which I discovered by accident. The file uses `CUB_DEPRECATED_BECAUSE` which is defined in `<cub/util_deprecated.cuh>`